### DISCUSSION TITLE
Include promptfoo tests

### DISCUSSION THREAD

**Discussion by ghost** [spdustin/ChatGPT-AutoExpert#53] [Ideas]
I noticed in the substack post for AutoExpert you mentioned using promptfoo to evaluate the prompts. I think it would be useful to include these tests in the repo so users can see how the prompts are evaluated, and evaluate their own changes.

**Comment by spdustin** [comment]
Indeed, I agree! I’ve been re-writing them for an upcoming Substack post on evaluations (along with a new edition to AutoExpert and updates to the existing editions), and I’ll be including PromotFoo tests in that update!
